client/signup/accordion-form/accordion-form-section.tsx

```diff
@@ -22,6 +22,9 @@ interface SectionHeaderProps {
 const Section = styled.div`
 	border-bottom: 1px solid var( --studio-gray-5 );
 	padding: 0 20px;
+	@media ( min-width: 600px ) {
+		width: 775px;
+	}
```

Comment:
Can you please help me understand why this change was required? 🤔

Reply:
This is tied to the flex wrap. By providing a concrete width I am forcing the image upload placeholders to wrap to the next line beyond a certain width. :)